lab-javascript-clue/cluedo.js

```diff
@@ -0,0 +1,72 @@
+
+var suspectList = [mrGreen = {
+  name: "Jacob Green",
+  colour: "green",
+  occupation: "unknown",
+  age: undefined,
+  description: "He has a lot of connections and is always willing to help people out -- for a price."
+}, drOrchid = {name: "Doctor Orchid"}, victor = {name: "Victor Plum"},
+kasandra = {name: "Kasandra Scarlet"}, eleanor = {name: "Eleanor Peacock"},
+jack = {name: "Jack Mustard"}
+];
+
+var weaponList = [ rope = {
+  name: "rope",
+  material: "cotton",
+  color: "dirty white",
+  dimensions: "2 meters",
+  hurtFactor: 1,
+  weight: "light"
+}, knife = {name: "knife"}, candlestick = {name: "candlestick"},
+dumbbell = {name: "dumbbell"}, poison = {name: "poison"},
+axe = {name: "axe"}, bat = {name: "bat"}, trophy = {name: "trophy"},
+pistol = {name: "pistol"}
+];
+
+
+var roomsList = [ hall = {
+  sqrMeters: 20,
+  windows: true,
+  locked: false
+}, diningRoom = {name: "diningRoom"}, conservatory = {name: "conservatory"},
+kitchen = {name: "kitchen"}, study = {name: "study"}, library = {name: "library"},
+billiardRoom = {name: "billiardRoom"}, lounge = {name: "lounge"},
+ballRoom = {name: "ballRoom"}, spa = {name: "spa"}, livingRoom = {name: "livingRoom"},
+observatory = {name: "observatory"}, theatre = {name: "theatre"},
+guestHouse = {name: "guestHouse"}, patio = {name: "patio"}
+];
+
+
+// Method to randomly select one element from a card stack (General)
+function randomCard(cardList) {
+  var min = 0;
+  var max = cardList.length - 1;
+  var randNum = Math.floor(Math.random() * (max - min));
+  var card = cardList[randNum];
+  return card;
+};
```

Comment:
`var max = cardList.length - 1`;   should be `var max = cardList.length`
otherwise you are leaving out the last item in the array.
Since you know the min value is gonna be zero, you could refactor as: 
`var randNum = Math.floor(Math.random() * cardList.length);`